openupgrade_scripts/scripts/hr/16.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,75 @@
+---Models in module 'hr'---
+model hr.contract.type (moved from hr_contract)
+# DONE: pre-migration: moved from hr_contract to hr
+
+---Fields in module 'hr'---
+hr           / hr.contract.type         / __last_update (datetime)      : previously in module hr_contract
+hr           / hr.contract.type         / _order                        : previously in module hr_contract
+hr           / hr.contract.type         / display_name (char)           : previously in module hr_contract
+hr           / hr.contract.type         / name (char)                   : previously in module hr_contract
+# DONE: pre-migration: moved from hr_contract to hr
+
+hr           / hr.contract.type         / sequence (integer)            : NEW
+# NOTHING TO DO
+
+hr           / hr.department            / master_department_id (many2one): NEW relation: hr.department, isfunction: function, stored
+hr           / hr.department            / parent_path (char)            : NEW
```

Comment:
@duong77476 chỗ này cần test xem parent_path có giá trị không, nếu không thì cần điền giá trị cho parent_path và master_department_id

Reply:
Khả năng là không rồi anh ạ, vì 15 không có ông `parent_path` này

Reply:
à chưa chắc vì nó sẽ tính toán `parent_path` ở hàm `auto_init`, này để e test lại phát

Reply:
E test thì nó có điền luôn `parent_path` khi migrate rồi anh ạ, e quên ko cho vào video test, e gửi cái ảnh ở đây thôi

![Screenshot from 2023-07-21 11-45-48](https://github.com/Viindoo/OpenUpgrade/assets/56789189/e4bce6c1-881b-4ff1-b8aa-e920ad1585ac)